tests/components/markers/marker-test.js

```diff
@@ -0,0 +1,112 @@
+jest
+  .dontMock('../../../js/components/markers/marker')
+  .dontMock('classnames');
+
+import React from 'react';
+import Enzyme from 'enzyme';
+import Marker from '../../../js/components/markers/marker';
+
+describe('Marker component', function () {
+
+  let baseProps;
+
+  function renderComponent() {
```

Comment:
Please don't use a mutable object from closure, pass props directly to this function. Otherwise, it's hard to follow what's going on

Reply:
Thanks, good catch!

Reply:
Done. I made a few changes to pass a new set of props to that function.